docs/DeveloperGuide.md

```diff
@@ -343,6 +362,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: UC03 - Add volunteer hours**
+
+**MSS**
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to add some number of volunteer hours to a specific person in the list
+4.  AddressBook adds the specified number of volunteer hours to the specified person's existing hours
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. User could also search for persons by name/tag/role... etc. (UC01)
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+
+**Use case: UC04 - Get emails**
+
+**MSS**
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to get emails of persons on the list
+4.  AddressBook copies to the user's clipboard a list of all emails of persons on the list
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. User could also search for persons by name/tag/role... etc. (UC01)
+
+* 2a. The list is empty.
+
+  Use case ends.
```

Comment:
![image](https://github.com/user-attachments/assets/3cdc02e4-7d46-47e9-acd0-940f0740994f)
Same formatting query about the extension